packages/sui-lint/.prettierrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   /* Support optional chaining new ES feature -> ? */
   optionalChaining: true,
   /* Specify the line length that the printer will wrap on. */
-  printWidth: 80,
+  printWidth: 120,
```

Comment:
Can we set it to 200? or 300? 😂 (cc @carlosvillu)

Reply:
300 even better !!!

Reply:
it's a 50% increase already but I agree... even 120 characters may feel short.